src/status_im/subs/wallet/wallet.cljs

```diff
@@ -126,18 +126,23 @@
  :<- [:wallet/wallet-send]
  :-> :route)
 
+(rf/reg-sub
+ :wallet/wallet-send-enough-assets?
+ :<- [:wallet/wallet-send]
+ :-> :enough-assets?)
+
 (rf/reg-sub
  :wallet/wallet-send-token
  :<- [:wallet/wallet-send]
  :<- [:wallet/network-details]
  :<- [:wallet/wallet-send-disabled-from-chain-ids]
  (fn [[wallet-send networks disabled-from-chain-ids]]
-   (let [token                  (:token wallet-send)
-         enabled-from-chain-ids (->> networks
-                                     (filter #(not (contains? (set disabled-from-chain-ids)
-                                                              (:chain-id %))))
-                                     (map :chain-id)
-                                     set)]
+   (let [token                    (:token wallet-send)
+         disabled-from-chain-ids? (set disabled-from-chain-ids)
+         enabled-from-chain-ids   (->> networks
+                                       (map :chain-id)
+                                       (remove disabled-from-chain-ids?)
+                                       set)]
```

Comment:
This is just a code style refactor, it does the same.

with this refactor:
1. We avoid calling `set` on the same `disabled-from-chain-ids` on each iteration, now it's called once
2. `filter ` + `not` -> `remove`
3. use the advante of the data-type being a set, so we don't need to call `contains?`